src/apps/concepts/pages/ViewConceptsPage.tsx

```diff
@@ -381,7 +396,8 @@ const mapActionsToProps = {
   retrieveDictionary: makeRetrieveDictionaryAction(true),
   retrieveSource: retrieveSourceAndDetailsAction,
   addConceptsToDictionary: recursivelyAddConceptsToDictionaryAction,
-  removeConceptsFromDictionary: removeReferencesFromDictionaryAction
+  removeConceptsFromDictionary: removeReferencesFromDictionaryAction,
+  retrieveSources: retrievePublicSourcesAction
```

Comment:
I would reuse the `retrievePublicSources` API that already exists in `src/apps/concepts/api.ts` for this. Despite it name, it actually will get all of the sources the current user has access to. This is the same thing we use to get the sources in the `MappingsTableRow.tsx` file.

It actually might be a good idea to wrap this up in it's own Redux action and Thunk so that we can cache the results somewhere instead of needing to load this list everytime, but that doesn't need to be part of this ticket.

I haven't tried this out, but it looks like it does the right things.

Reply:
> It actually might be a good idea to wrap this up in it's own Redux action and Thunk so that we can cache the results somewhere instead of needing to load this list everytime, but that doesn't need to be part of this ticket.
> 
> I haven't tried this out, but it looks like it does the right things.

Is it okay to put it the the follow up PR of this ticket https://issues.openmrs.org/browse/OCLOMRS-967?